src/backend/mi2/mi2.ts

```diff
@@ -763,13 +758,13 @@ export class MI2 extends EventEmitter implements IBackend {
 			this.process.stdin.write(raw + "\n");
 	}
 
-	async sendCliCommand(command: string, threadId: number = 0, frameLevel: number = 0) {
+	sendCliCommand(command: string, threadId: number = 0, frameLevel: number = 0): Thenable<MINode> {
```

Comment:
Changing this from async to direct call seems like a bigger change, can we move that to a separate PR?

Reply:
The reason I had to change that was to make the sendCliCommand and sendCommand both have the same type signature.  This is because sendCliCommand is now used in the MI2_LLDB.initCommands to send the extra commands (i.e., "settings append target.source-map" command).  All of the command promises are collected together and thus it seemed to be consistent to combine the commands sent with sendCliCommand together with those sent with sendCommand.

Reply:
I see. @WebFreak001 Do you see any issues with that or can we pull?
"Kind of related": Could you include a [GH Action to publish the extension](https://github.com/HaaLeo/publish-vscode-extension#readme)? This would allow people with appropriate rights to get an update out of the door without the need for you to join and manually handle that.

Reply:
changing from async to returning the Thenable should have no effect other than the return value being returned now, so fine with that.

Publishing automatically sounds like a good idea, debugging the CI if there is a problem will be a pain though